core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -123,6 +123,20 @@ public void run() {
       if (config.isAccessLogsEnabled()) {
         routerFactory.addGlobalHandler(LoggerHandler.create(LoggerFormat.DEFAULT));
       }
+
+      // security handler for keymanager endpoints
+      routerFactory.addSecurityHandler(
```

Comment:
Can this be moved into the Eth2Runner as this only applies for Eth2

Reply:
done